core/primitives/res/runtime_configs/parameters.txt

```diff
@@ -0,0 +1,152 @@
+# Gas economics config
+burnt_gas_reward_numerator: 3
+burnt_gas_reward_denominator: 10
+pessimistic_gas_price_inflation_numerator: 103
+pessimistic_gas_price_inflation_denominator: 100
+
+# Account creation config
+min_allowed_top_level_account_length: 32
+registrar_account_id: registrar
+
+# Storage usage config
+storage_amount_per_byte: 100000000000000000000
+storage_num_bytes_account: 100
+storage_num_extra_bytes_record: 40
+
+# Static action costs:
+# send_sir / send_not_sir is burned when creating a receipt on the signer shard
+# (SIR = signer is receiver, which guarantees the receipt is local)
+# execution is burned when applying receipt on receiver shard
+action_receipt_creation_send_sir: 108059500000
+action_receipt_creation_send_not_sir: 108059500000
+action_receipt_creation_execution: 108059500000
+data_receipt_creation_base_send_sir: 4697339419375
```

Comment:
Can we format numbers with thousand separators? `4_697_339_419_375`

Reply:
Done